controllers/topics.controllers.js

```diff
@@ -10,4 +12,8 @@ exports.getTopics = (req, res, next) => {
         console.log(err)
         next(err)
     })
-};
\ No newline at end of file
+};
+
+exports.getApi = (req, res) => {
+    res.status(200).send(endPoints);
```

Comment:
We should be sending data back in an object with a key - {data: endpoints}, for example